worlds/tunic/options.py

```diff
@@ -187,6 +166,98 @@ class TunicPlandoConnections(PlandoConnections):
+    """
+    internal_name = "combat_logic"
+    display_name = "More Combat Logic"
+    option_off = 0
+    option_bosses_only = 1
+    option_on = 2
+    default = 0
+
+
+class LaurelsZips(Toggle):
+    """
+    Choose whether to include using the Hero's Laurels to zip through gates, doors, and tricky spots.
+    Notable inclusions are the Monastery gate, Ruined Passage door, Old House gate, Forest Grave Path gate, and getting from the Back of Swamp to the Middle of Swamp.
+    """
+    internal_name = "laurels_zips"
+    display_name = "Laurels Zips Logic"
+
+
+class IceGrappling(Choice):
+    """
+    Choose whether grappling frozen enemies is in logic.
+    Easy includes ice grappling enemies that are in range without luring them. May include clips through terrain.
+    Medium includes using ice grapples to push enemies through doors or off ledges without luring them. Also includes bringing an enemy over to the Temple Door to grapple through it.
+    Hard includes luring or grappling enemies to get to where you want to go.
+    The Medium and Hard options will give the player the Torch to return to the Overworld checkpoint to avoid softlocks.
+    Note: You will still be expected to ice grapple to the slime in East Forest even with this option off.
+    """
+    internal_name = "ice_grappling"
+    display_name = "Ice Grapple Logic"
+    option_off = 0
+    option_easy = 1
+    option_medium = 2
+    option_hard = 3
+    default = 0
+
+
+class LadderStorage(Choice):
+    """
+    Choose whether Ladder Storage is in logic.
+    Easy includes uses of Ladder Storage to get to open doors over a long distance without too much difficulty. May include convenient elevation changes (going up Mountain stairs, stairs in front of Special Shop, etc.).
+    Medium includes the above as well as changing your elevation using the environment and getting knocked down by melee enemies mid-LS.
+    Hard includes the above as well as going behind the map to enter closed doors from behind, shooting a fuse with the magic wand to knock yourself down at close range, and getting into the Cathedral Secret Legend room mid-LS.
+    Enabling any of these difficulty options will give the player the Torch item to return to the Overworld checkpoint to avoid softlocks. Using the Torch is considered in logic.
+    Opening individual chests while doing ladder storage is excluded due to tedium.
+    Knocking yourself out of LS with a bomb is excluded due to the problematic nature of consumables in logic.
+    """
+    internal_name = "ladder_storage"
+    display_name = "Ladder Storage Logic"
+    option_off = 0
+    option_easy = 1
+    option_medium = 2
+    option_hard = 3
+    default = 0
+
+
+class LadderStorageWithoutItems(Toggle):
+    """
+    If disabled, you logically require Stick, Sword, or Magic Orb to Ladder Storage.
+    If enabled, you will be expected to Ladder Storage without progression items.
```

Comment:
Probably should say "use"?
```suggestion
    If disabled, you logically require Stick, Sword, or Magic Orb to use Ladder Storage.
    If enabled, you will be expected to use Ladder Storage without progression items.
```

Reply:
I'm using Ladder Storage as a verb here. I'll run it by some others in the Tunic community and see what they think